pkg/devicerepository/store/profiles.go

```diff
@@ -125,7 +125,7 @@ func (p EndDeviceProfile) ToTemplatePB(ids *ttnpb.EndDeviceVersionIdentifiers, i
 		paths = append(paths, "mac_settings.ping_slot_data_rate_index")
 	}
 	if p.PingSlotFrequency > 0 {
```

Comment:
This is not ideal because it does not differentiate between 'unknown' and 0. It would have been great if this was nullable in the device repository I think.